Program.cs

```diff
@@ -182,6 +182,21 @@ public static void Main(string[] args)
                     // package for distribution
                     runCommand("ditto", $"-ck --rsrc --keepParent --sequesterRsrc {stagingPath}/osu!.app {releasesPath}/osu!.app.zip");
                     break;
+                case RuntimeInfo.Platform.Linux:
+
+                    // need to add --self-contained flag for AppImage distribution.
+                    runCommand("dotnet", $"publish -r linux-x64 --self-contained --configuration Release -o {stagingPath}/osu.AppDir  /p:Version={version}");
+
+
+                    runCommand("chmod", $"-R 755 {stagingPath}/osu.AppDir/*");
+

```

Comment:
empty line

Reply:
Following spacing conventions from the case above.

Reply:
You have 2 empty lines in a few places instead of the usual one.

Reply:
@Aergwyn yeah I saw it, I was inconsistent with spacing. Will Fix